charts/timed/templates/service-postgresql.yaml

```
@@ -0,0 +1,22 @@
+{{- if .Values.postgresql.ingress.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: postgresql-ingress
+  labels:
+    {{- include "timed.labels" . | nindent 4 }}
+    app: postgresql
+spec:
+  type: LoadBalancer
+  ports:
+    - port: 5432
+      targetPort: 5432
+      protocol: TCP
+      name: tcp-postgresql
+  selector:
+    app: postgresql
+    role: slave
+  {{- if .Values.postgresql.ingress.ipWhitelist }}
+  loadBalancerSourceRanges: {{ .Values.postgresql.ingress.ipWhitelist | toYaml | nindent 4 }}
+  {{- end }}
```

I'm not sure if the nindent on the same line as the value will work.
```suggestion
  {{- if .Values.postgresql.ingress.loadBalancerSourceRanges }}
  loadBalancerSourceRanges: 
    {{ .Values.postgresql.ingress.loadBalancerSourceRanges | toYaml | nindent 4 }}
  {{- end }}
```